Zend/tests/arginfo_zpp_mismatch.phpt

```diff
@@ -4,10 +4,83 @@ Test that there is no arginfo/zpp mismatch
 <?php
 
 function test($function) {
+    if (false
```

Comment:
Hmm did you miss to remove this hardcoded `false`?

Reply:
I think this is deliberate for code formatting purposes.